src/webui/src/components/PackageSidebar/modules/Infos/index.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import get from 'lodash/get';
+import Module from '../../Module';
+import isString from 'lodash/isString';
+import isNil from 'lodash/isNil';
+
+import classes from './style.scss';
+
+export default class Infos extends React.Component {
+  static propTypes = {
+    packageMeta: PropTypes.object.isRequired
+  };
+
+  get infos() {
+    const homepage = this.normalizeInfo(get(this, 'props.packageMeta.latest.homepage', null));
+    const repo = this.normalizeInfo(get(this, 'props.packageMeta.latest.repository', null));
+    const license = get(this, 'props.packageMeta.latest.license', 'N/A');
+
+    return {homepage, repo, license};
+  }
+
+  normalizeInfo(infoObj) {
+    if (isString(infoObj)) {
+      return {url: infoObj};
+    } else if (isNil(infoObj)) {
+      return {url: ''};
+    }
+
+    infoObj.url = this.normalizeGitUrl(infoObj);
+
+    return infoObj;
+  }
+
+  normalizeGitUrl(infoObj) {
+    return infoObj.url.replace(/^git\+/, '');
+  }
+
+  render() {
+    const infos = this.infos;
+
+    if (infos.homepage.url === '' && infos.repo.url === '' && infos.license === 'N/A') {
+      return '';
```

Comment:
Just wondering, what happens whether the repository has no license but has homepage and url? Do you want to do not display at least what is available? What do you think?

Reply:
Nevermind ! I did not see conditions bellow.